src/controllers/RootController.ts

```diff
@@ -1,15 +1,13 @@
 import express from 'express';
-import {
-  Body,
-  Controller, Get, Post, Query, Request, Response, Route, Security,
-} from 'tsoa';
+// eslint-disable-next-line @typescript-eslint/no-redeclare
```

Comment:
Why is this ESLint ignore here? Please try to remove it

Reply:
It's about the import of Body, Request and Response from tsoa. These names are already built-in global variables, but it is not possible to rename the tsoa imports. I've had this problem in a different project too. As far as I know, there is no fix other than a local or global ESLint ignore.

Reply:
You can do `import { X as Y } from 'Z';` to import something under a different name.